sample/build.gradle.kts

```diff
@@ -90,6 +91,10 @@ sourceSets {
 
 tasks.withType<KotlinCompile> {
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = "17"
```

Comment:
It shouldn't need a version update. Keep on the same version.

Reply:
I have downgraded it back to 1.8